.github/workflows/publish-oss-for-cloud.yml

```diff
@@ -1,6 +1,8 @@
 name: Publish OSS Artifacts for Cloud
-concurrency:
-  group: ${{ github.workflow }}-${{ inputs.oss_ref || github.sha }}
+
+env:
+  S3_BUILD_CACHE_ACCESS_KEY_ID: ${{ secrets.SELF_RUNNER_AWS_ACCESS_KEY_ID }}
```

Comment:
Maybe a comment is nice here that says something along the lines of 

//these environment variables are necessary for gradle cache task